art/attacks/poisoning/hidden_trigger_backdoor/hidden_trigger_backdoor_pytorch.py

```diff
@@ -240,6 +240,8 @@ def poison(  # pylint: disable=W0221
 
                 for _ in range(feat2.size(0)):
                     dist_min_index = (dist == torch.min(dist)).nonzero().squeeze()
+                    if dist_min_index.dim() > 1:  # If multiple values in dist equal torch.min(dist), return the first.
```

Comment:
There could be several items in `dist ` with the same minimum value. Hence, `dist_min_index`, rather than being an expected index, is a tensor with several values.  To avoid this crashing the attack, we pick the first returned index. This bug was discovered when adapting the attacks to run with HF.